src/algorithm/retrieveAccessTokenIssuerKeySet.ts

```diff
@@ -1,5 +1,8 @@
-import { fetch as crossFetch } from "cross-fetch";
+// eslint-disable-next-line no-shadow
```

Comment:
I don't fully understand why this rule pops up so many times in this PR. You could also rename the classes that you import so that you don't have to disable it but this doesn't look like something you would have to disable in the first place.

Reply:
Fixed the fetch shadow issue here: https://github.com/solid/access-token-verifier/pull/24/commits/d0d288d5c46ecd6d3fa5a434a9f7a84c4840c419

Just needed to target the node env and not browser in eslint.

(still investigating what is happening with URL)

Reply:
Failed attempt at resolving URL no-shadow in https://github.com/solid/access-token-verifier/pull/24/commits/57769c93f97c83a21edf089e4dc53aee3c1751c6

The problem is that we don't want the DOM types in TypeScript and on the other hand, the eslint no-shadow rule doesn't seem to currently apply correctly its "allow" directive (when I enable it, nothing that's shadowed gets picked up).